packages/twenty-front/src/modules/ui/input/editor/components/CustomSideMenuOptions.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { IconComponent } from 'twenty-ui';
+
+const StyledContainer = styled.div<{ variant: variants }>`
+  color: ${({ theme, variant }) =>
+    variant === 'Danger' ? theme.color.red : 'inherit'};
```

Comment:
Usually no capital letter for variant I think? `danger`

Reply:
Will check with other components and make the necessary changes

Reply:
It is indeed `danger`, made the changes